app/views/user_mailer/batch_subscription_notification.text.erb

```diff
@@ -3,11 +3,12 @@
 <% @seen_summary = {} %>
 <% @creations.each_with_index do |creation, index| %>
 <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
-<%= this_work.pseuds.map{|p| p.byline}.to_sentence.html_safe %> posted a <% if creation.is_a?(Work) then %><%= this_work.backdate ? "backdated" : "new" %> work:<% else %>new chapter of <%= this_work.title %> (<%= this_work.word_count %> words):<% chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe %><% end %>
+<% creators = creator_links(this_work) %>
```

Comment:
I think what we're aiming for up here is just a list of bylines, e.g. `user1, User 2 (user2), and user3 posted a new work:`, and below it should be the user's byline followed by the pseud's URL in parentheses, e.g. `by user1 (http://archiveofourown.org/users/user1/pseuds/user1), User 2 (user2)  (http://archiveofourown.org/users/user2/pseuds/User%202), and user3 (http://archiveofourown.org/users/user3/pseuds/user3)`.

However, it looks like `creator_links` uses the `style_pseud_link` helper, which involves a lot of HTML and CSS we probably don't want in plain text emails. Is it getting stripped out somewhere?